docs/migration_guides/opflow_migration.rst

```diff
@@ -0,0 +1,1748 @@
+The function equivalency can be roughly summarized as follows:
+
+.. |operator_globals| replace:: ``operator_globals``
+.. _operator_globals: https://qiskit.org/documentation/apidoc/opflow.html#operator-globals/
+
+.. list-table::
+   :header-rows: 1
+
+   * - Opflow Module
+     - Alternative
+   * - Operators (:class:`~qiskit.opflow.OperatorBase`, |operator_globals|_ ,
+       :mod:`~qiskit.opflow.primitive_ops`,
+       :mod:`~qiskit.opflow.list_ops`\)
+     - :mod:`qiskit.quantum_info` :ref:`quantum_info_operators`
+
+   * - :mod:`qiskit.opflow.state_fns`
+     - :mod:`qiskit.quantum_info` :ref:`quantum_info_states`
+
+   * - :mod:`qiskit.opflow.converters`
+     - :mod:`qiskit.primitives`
+
+   * - :mod:`qiskit.opflow.evolutions`
+     - :mod:`qiskit.quantum_info` :ref:`quantum_info_synthesis`
+
+   * - :mod:`qiskit.opflow.expectations`
+     - :class:`qiskit.primitives.Estimator`
+
+   * - :mod:`qiskit.opflow.gradients`
+     - :mod:`qiskit.algorithms.gradients`
+
+.. |qiskit_aer.primitives| replace:: ``qiskit_aer.primitives``
+.. _qiskit_aer.primitives: https://qiskit.org/documentation/locale/de_DE/apidoc/aer_primitives.html
+
+.. |qiskit_aer.primitives.Sampler| replace:: ``qiskit_aer.primitives.Sampler``
+.. _qiskit_aer.primitives.Sampler: https://qiskit.org/documentation/locale/de_DE/stubs/qiskit_aer.primitives.Sampler.html
+
+.. |qiskit_aer.primitives.Estimator| replace:: ``qiskit_aer.primitives.Estimator``
+.. _qiskit_aer.primitives.Estimator: https://qiskit.org/documentation/locale/de_DE/stubs/qiskit_aer.primitives.Estimator.html
+
+.. |qiskit_ibm_runtime| replace:: ``qiskit_ibm_runtime``
+.. _qiskit_ibm_runtime: https://qiskit.org/documentation/partners/qiskit_ibm_runtime/primitives.html
+
+.. |qiskit_ibm_runtime.Sampler| replace:: ``qiskit_ibm_runtime.Sampler``
+.. _qiskit_ibm_runtime.Sampler: https://qiskit.org/documentation/partners/qiskit_ibm_runtime/stubs/qiskit_ibm_runtime.Sampler.html
+
+.. |qiskit_ibm_runtime.Estimator| replace:: ``qiskit_ibm_runtime.Estimator``
+.. _qiskit_ibm_runtime.Estimator: https://qiskit.org/documentation/partners/qiskit_ibm_runtime/stubs/qiskit_ibm_runtime.Estimator.html
+
+..  attention::
+
+    Most references to the :class:`qiskit.primitives.Sampler` or :class:`qiskit.primitives.Estimator` in this guide
+    can be replaced with instances of the:
+
+    - Aer primitives (|qiskit_aer.primitives.Sampler|_, |qiskit_aer.primitives.Estimator|_)
+    - Runtime primitives (|qiskit_ibm_runtime.Sampler|_, |qiskit_ibm_runtime.Estimator|_ )
+    - Terra backend primitives (:class:`qiskit.primitives.BackendSampler`, :class:`qiskit.primitives.BackendEstimator`)
+
+    Certain classes, such as the
+    :class:`~qiskit.opflow.expectations.AerPauliExpectation`, can only be replaced by a specific primitive instance
+    (in this case, |qiskit_aer.primitives.Estimator|_ ), or require a specific option configuration.
+    If this is the case, it will be explicitly indicated in the corresponding section.
+
+Contents
```

Comment:
I wonder whether we should move this section to above `Background` and below `TD;DR`.

Reply:
Makes sense